src/subcommand/giraffe_main.cpp

```diff
@@ -1776,8 +1808,19 @@ int main_giraffe(int argc, char** argv) {
         cerr << "Loading Minimizer Index" << endl;
     }
     unique_ptr<gbwtgraph::DefaultMinimizerIndex> minimizer_index;
+    unique_ptr<gbwtgraph::MinimizerIndexXL> path_minimizer_index(new gbwtgraph::MinimizerIndexXL());
     if (map_long_reads) {
-        minimizer_index = vg::io::VPKG::load_one<gbwtgraph::DefaultMinimizerIndex>(registry.require("Long Read Minimizers").at(0));
+        if (use_path_minimizer) {
+            // FIXME: This is a temporary hack to load the path minimizers, I can't find a way to use VPKG to load them.
```

Comment:
You would need to register the type with the VPKG system in a `.cpp` file in `src/io/` in vg, and I think it would also need to have a different magic number than the other minimizer index version.